test_pr_quick_start_guide.rst

```diff
@@ -0,0 +1,91 @@
+*****************************
+Quick-start development guide
+*****************************
+
+This guide describes all steps needed to test another author's pull request (hereinafter ``PR``).
```

Comment:
```suggestion
This guide describes all steps needed to test another author's pull request (hereinafter ``PR``).

It can be useful, for example, for reviewers and issue authors when they want to check if a PR fixes the reported bug.
```

Reply:
I've done a minor change in your sentence in https://github.com/ansible/community-docs/pull/16/commits/78854b70fd32d02aca4688f443dbda61386d2b27.